qiskit/pulse/macros.py

```diff
@@ -16,12 +16,14 @@
 
 from qiskit.pulse import channels, exceptions, instructions, utils
 from qiskit.pulse.instruction_schedule_map import InstructionScheduleMap
-from qiskit.pulse.schedule import Schedule
+from qiskit.pulse.schedule import Schedule, ScheduleBlock
+from qiskit.transpiler.target import Target
 
 
 def measure(
```

Comment:
Can you start _pending_ deprecation for `inst_map` and V1 backend? We have a convenient decorator now:
https://github.com/Qiskit/qiskit-terra/blob/58aceb59e9bc1e16f29b09402e1984b16d211da3/qiskit/utils/deprecation.py#L100-L111

Reply:
Should I move `deprecate_arg` part to `_measure_v1`?